scripts/ERC20ForwardersDeployerKovan.js

```diff
@@ -102,47 +140,66 @@ async function main() {
   
     console.log('✅ USDT support added');
     console.log(`✅ USDT address : ${usdtAddress}`);
+    console.log(`gas used : ${receipt.gasUsed.toNumber()}`);
+    totalGasUsed = totalGasUsed + receipt.gasUsed.toNumber();
   
     tx = await forwarderProxy.setOracleAggregator(oracleAggregator.address);
     receipt = await tx.wait(priceFeedUsdc = await hre.ethers.getContractAt("AggregatorInterface", usdcEthPriceFeedAddress));  
     console.log(`✅ Oracle aggregator added`);
+    console.log(`gas used : ${receipt.gasUsed.toNumber()}`);
+    totalGasUsed = totalGasUsed + receipt.gasUsed.toNumber();
   
     //set transfer handler gas
     tx = await forwarderProxy.setTransferHandlerGas(daiAddress, DaiTransferHandlerGas); //values to be tuned further
-    receipt = await tx.wait(confirmations = 2);
-  
+    receipt = await tx.wait(confirmations = 2); 
     console.log(`✅ DAI transfer handler gas ${DaiTransferHandlerGas} added`);
+    console.log(`gas used : ${receipt.gasUsed.toNumber()}`);
+    totalGasUsed = totalGasUsed + receipt.gasUsed.toNumber();
+
     tx = await forwarderProxy.setTransferHandlerGas(usdcAddress, USDCTransferHandlerGas);
-    receipt = await tx.wait(confirmations = 2);
-  
+    receipt = await tx.wait(confirmations = 2);  
     console.log(`✅ USDC transfer handler gas ${USDCTransferHandlerGas} added`);
+    console.log(`gas used : ${receipt.gasUsed.toNumber()}`);
+    totalGasUsed = totalGasUsed + receipt.gasUsed.toNumber();
 
     tx = await forwarderProxy.setTransferHandlerGas(usdtAddress, USDTransferHandlerGas);
-    receipt = await tx.wait(confirmations = 2);
-  
+    receipt = await tx.wait(confirmations = 2); 
     console.log(`✅ USDT transfer handler gas ${USDTransferHandlerGas} added`);
+    console.log(`gas used : ${receipt.gasUsed.toNumber()}`);
+    totalGasUsed = totalGasUsed + receipt.gasUsed.toNumber();
 
     tx = await forwarderProxy.setSafeTransferRequired(usdtAddress,true);
     receipt = await tx.wait(confirmations = 2);
     console.log(`✅ USDT is marked for safe transfer`);
+    console.log(`gas used : ${receipt.gasUsed.toNumber()}`);
```

Comment:
Follow standard case.. start the sentence with capital letter.
eg. 
`Gas used : ${receipt.gasUsed.toNumber()}`

Reply:
done.